_config.yml

```diff
@@ -73,6 +73,22 @@ FLINK_TABLE_STORE_GITHUB_REPO_NAME: flink-table-store
 #          md1_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics-prometheus_2.12/1.7.1/flink-metrics-prometheus_2.12-1.7.1.jar.sha1
 
 flink_releases:
+  - - version_short: "1.16"
+      binary_release:
+        name: "Apache Flink 1.16.0"
+        scala_212:
+          id: "1160-download_212"
+          url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.16.0/flink-1.16.0-bin-scala_2.12.tgz"
```

Comment:
After visiting the URL for [1.15.2](https://www.apache.org/dyn/closer.lua/flink/flink-1.15.2/flink-1.15.2-bin-scala_2.12.tgz)([blog-link](https://flink.apache.org/downloads.html#apache-flink-1152)), the page says
```
We suggest the following site for your download:

https://dlcdn.apache.org/flink/flink-1.15.2/flink-1.15.2-bin-scala_2.12.tgz
```
not sure if we need to update the link here.

Reply:
no, the above url will jump to the download page, which has the above content. And other urls also start with `https://www.apache.org/`